contracts/protocol/lendingpool/MOO/MOOConfiguratorAlfajores.sol

```diff
@@ -0,0 +1,93 @@
+pragma solidity 0.6.12;
+pragma experimental ABIEncoderV2;
+
+import {ERC20} from '../../../dependencies/openzeppelin/contracts/ERC20.sol';
+import {Ownable} from '../../../dependencies/openzeppelin/contracts/Ownable.sol';
+import '../../../protocol/lendingpool/LendingPoolConfigurator.sol';
+import '../../../interfaces/ILendingPoolConfigurator.sol';
+
+contract MOOConfiguratorAlfajores is Ownable {
+  address constant lendingPoolConfiguratorAddress = 0x39fe2A4a4174bB5cAC5568ce0715a0b320bcB231;
+
+  LendingPoolConfigurator public lendingPoolConfigurator =
+    LendingPoolConfigurator(lendingPoolConfiguratorAddress);
+  address constant assetAddress = 0x17700282592D6917F6A73D0bF8AcCf4D578c131e;
+
+  bytes constant params = '0x10';
+  bool constant stableBorrowRateEnabled = true;
+  uint8 constant underlyingAssetDecimals = 18;
+  address constant aTokenImpl = 0xe8B286649713447D8d5fBeBC28c731830d19B6C9;
+  address constant stableDebtTokenImpl = 0xB6a5059A228a16Fa2827E28E52ceC96BBC63D639;
+  address constant variableDebtTokenImpl = 0xB65b6a6a6F78E4daABF259c756567ae346699687;
+  address constant interestRateStrategyAddress = 0x5B41b0c78659636c6664f08F7cCb620ceA3F1206;
```

Comment:
I guess interest rate strategy should be the same as CELO, because it is much closer in volatility to MOO.